ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/MixedTableMaintainer.java

```diff
@@ -107,6 +108,16 @@ public void expireSnapshots(TableRuntime tableRuntime) {
     baseMaintainer.expireSnapshots(tableRuntime);
   }
 
+  @Override
+  public void autoCreateTags(TableRuntime tableRuntime) {
+    new AutoCreateIcebergTagAction(
+            arcticTable.isKeyedTable()
+                ? arcticTable.asKeyedTable().baseTable()
+                : arcticTable.asUnkeyedTable(),
+            LocalDateTime.now())
+        .execute();
+  }
```

Comment:
If there is data in the Change store, there is a risk of data loss for keyed-tables.